helmfile/charts/networkpolicy-service/templates/harbor/allow-harbor-core.yaml

```
@@ -0,0 +1,171 @@
+{{ if .Values.harbor.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-harbor-core
+  namespace: harbor
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      component: "core"
+  ingress:
+    - ports:
+        - port: 8080
+          protocol: TCP
+  egress:
+    - to:
+      {{- if .Values.global.externalLoadBalancer }}
+        {{- range $ip := .Values.global.scIngress.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      {{- else if .Values.global.ingressUsingHostNetwork }}
+        {{- range $ip := .Values.global.scNodes.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      {{- else }}
+        - namespaceSelector:
```

Don't you also need to add [this](https://github.com/elastisys/compliantkubernetes-apps/pull/1349/files#diff-09c655a89fde669d4a28279e488666530397122a163f85710998d3f0d184f78dR56-R59)
```suggestion
        {{- range $IP := .Values.global.ingress.ips }}
        - ipBlock:
            cidr: {{ $IP }}
        {{- end }}
        - namespaceSelector:
```

---

Ah yes, that should still be there. Will fix that